bool execution_success
int32 n_pose
---
float64[] next_pose

